libdnf5/solv/pool.cpp

```diff
@@ -79,6 +79,71 @@ Pool::~Pool() {
 }
 
 
+/// Returns a temporrary object allocated by pool_alloctmpspace
```

Comment:
Spelling: temporrary -> temporary, also in `pool.hpp`

Reply:
Thank you